internal/core/metadata/rest_addressable.go

```diff
@@ -19,30 +19,37 @@ import (
 	"net/url"
 	"strconv"
 
+	"github.com/edgexfoundry/edgex-go/internal/core/metadata/operators/addressable"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/errorConcept"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
 	"github.com/edgexfoundry/go-mod-core-contracts/models"
 	"github.com/gorilla/mux"
-
-	types "github.com/edgexfoundry/edgex-go/internal/core/metadata/errors"
-	"github.com/edgexfoundry/edgex-go/internal/core/metadata/operators/addressable"
-	"github.com/edgexfoundry/edgex-go/internal/pkg/db"
 )
 
+// Global variables
+var addressableErrorConcept errorConcept.AddressableErrorConcept
+
 func restGetAllAddressables(w http.ResponseWriter, _ *http.Request) {
 	op := addressable.NewAddressableLoadAll(Configuration.Service, dbClient, LoggingClient)
 	addressables, err := op.Execute()
 	if err != nil {
-		switch err.(type) {
-		case types.ErrLimitExceeded:
-			http.Error(w, err.Error(), http.StatusRequestEntityTooLarge)
-		default:
-			http.Error(w, err.Error(), http.StatusInternalServerError)
-		}
+		httpErrorHandler.Handle(
+			w,
+			err,
+			[]errorConcept.ErrorConceptType{
```

Comment:
Where you have a single element slice, recommend you confine definition to a single line; i.e. `[]errorConcept.ErrorConceptType{ commonErrorConcept.RequestEntityTooLarge },`